static/css/perdiem.css

```diff
@@ -166,3 +166,11 @@ p.profile-total {
     padding: 2em;
     margin-top: 2em;
 }
+
+.apply-page {
+    padding: 2em;
+}
+
+.apply-button {
+    text-align: center;
+}
```

Comment:
nit: Add newline here. Sorry, it's confusing. Every file should end in a newline character but there shouldn't be multiple at the end of the page. In other words, the last line in the file should have nothing in it, but the line before should have some characters in it.